packages/ckeditor5-code-block/src/codeblockcommand.js

```diff
@@ -41,8 +55,11 @@ export default class CodeBlockCommand extends Command {
 	 *
 	 * @fires execute
 	 * @param {Object} [options] Command options.
+	 * @param {String|null} [options.language] Language picked from dropdown.
```

Comment:
```suggestion
	 * @param {String} [options.language] The code block language.
```

1. When setting `options.language` to `null`, the command will use the default language. So it does not have to be specified. So, for example, `{String}` will be enough.
	* `[` and `]` marks the argument as optional, so if an integrator does not specify the value, the command will still work.
1. The language option does not have to come from the dropdown. The integrator can just execute the command directly: `editor.execute( 'codeBlock', { language: 'php' } )`. 
    * We do not have to focus on where the value comes from but what it does.